tests/static/js/jqunit/jqUnit-binder-commonEventBindings.js

```diff
@@ -0,0 +1,128 @@
+/* globals fluid, jqUnit */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+
+    // Component to test support for common event bindings
+    fluid.defaults("gpii.tests.binder.commonEventBindings", {
+        gradeNames: ["gpii.tests.binder.base", "gpii.binder.bindMarkupEvents"],
+        bindings: {
+        },
+        selectors: {
+            inputButton: "#input-button",
+            inputKeydown: "#input-button",
+            paragraphClick: ".paragraph-click",
+            multipleEventInputButton: "#input-multiple-events-button"
+        },
+        markupEventBindings: {
```

Comment:
Also supply a test which shows that these handlers can be overridden cleanly

Reply:
Added a test where 1) Some are left as is, 2) some are overridden, and 3) some new one is introduced.